mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/table/CassandraMessageV1Table.java

```diff
@@ -21,9 +21,9 @@
 
 import static org.apache.james.mailbox.cassandra.table.CassandraMessageIds.MESSAGE_ID;
 
-public interface CassandraMessageTable {
+public interface CassandraMessageV1Table {
 
-    String TABLE_NAME = "message";
```

Comment:
This is strictly forbiden! Do not rename tables, the code would never be shipped to OP.LNG.COM